src/main/java/bridge/model/Position.java

```diff
@@ -0,0 +1,21 @@
+package bridge.model;
+
+public class Position {
+    private int value;
+
+    public Position(int value) {
+        this.value = value;
+    }
+
+    public int getValue() {
+        return value;
+    }
+
+    public void increment() {
+        value++;
+    }
+
+    public void reset(int value) {
+        this.value = value;
+    }
```

Comment:
재시작시 고정된 값으로 초기화 될 것 같은데 생성자와 더불어서 
Postion 생성자에 파라미터 없이 시작점으로 초기화 하는건 어떨까요? 

```suggestion
    public Position() {
        this.value = BRIDGE_INIT_NUMBER.getNumber();
    }
    
    public void reset() {
        this.value = BRIDGE_INIT_NUMBER.getNumber();
    }
```